src/draive/generation/model/lmm.py

```diff
@@ -21,13 +21,13 @@
 ]
 
 
-async def lmm_generate_model[Generated: DataModel](  # noqa: PLR0913, C901
+async def lmm_generate_model[Generated: DataModel](  # noqa: PLR0913, C901, PLR0912
     generated: type[Generated],
     /,
     *,
     instruction: Instruction | str,
     input: MultimodalContent | MultimodalContentElement,  # noqa: A002
-    schema_variable: str | None = None,
+    schema_injection: Literal["auto", "full", "simplified", "skip"] | None = None,
```

Comment:
We should change default value to `"auto"` to state explicitly what will happen when `None` is passed. Additionally, I'm not so sure whether we should mark this parameter as optional.

Reply:
right, we could change "None" to "auto" as default. I would like to keep a default value which will extend prompt automatically adding the schema and format details. For simple cases it is good enough and makes it simpler to use

Reply:
I still don't know what's your answer.

> I would like to keep a default value which will extend prompt automatically adding the schema and format details

100% agree but I don't see this behaviour by looking at the method declaration.

Reply:
that is intended 😉 If you don't care about the details and it works fine it does not matter, just ask for model - when you will need to specify details you can then look for a parameter and use it